Documentation/platforms/xtensa/esp32/boards/esp32-devkitc/index.rst

```diff
@@ -83,3 +176,128 @@ outputted::
 
 From the host the message :code:`test` should be outputted.
 
+smp
+---
+
+Another NSH configuration, similar to nsh, but also enables
+SMP operation.  It differs from the nsh configuration only in these
+additional settings:
+
+SMP is enabled::
+
+  CONFIG_SMP=y
+  CONFIG_SMP_IDLETHREAD_STACKSIZE=3072
+  CONFIG_SMP_NCPUS=2
+  CONFIG_SPINLOCK=y
+
+The apps/testing/smp test is included::
+
+  CONFIG_TESTING_SMP=y
+  CONFIG_TESTING_SMP_NBARRIER_THREADS=8
+  CONFIG_TESTING_SMP_PRIORITY=100
+  CONFIG_TESTING_SMP_STACKSIZE=2048
+
+ostest
+------
+
+This is the NuttX test at apps/examples/ostest that is run against all new
+architecture ports to assure a correct implementation of the OS.  The default
+version is for a single CPU but can be modified for an SMP test by adding::
+
+  CONFIG_SMP=y
+  CONFIG_SMP_IDLETHREAD_STACKSIZE=2048
+  CONFIG_SMP_NCPUS=2
+  CONFIG_SPINLOCK=y
+
+NOTES:
+
+  1. See NOTES for the nsh configuration.
+  2. 2016-12-23: Test appears to be fully functional in the single CPU mode.
+  3. 2016-12-24: But when SMP is enabled, there is a consistent, repeatable
+     crash in the waitpid() test.  At the time of the crash, there is
+     extensive memory corruption and a user exception occurs (cause=28).
```

Comment:
ostest runs fine on SMP now.

Reply:
I'll remove it. Yeah, really don't like embedding these kind of notes in the docs. I would just have an open issue for it.

Reply:
I removed these three notes altogether. No point in having them.